gateway/proxy.go

```diff
@@ -28,6 +29,12 @@ var defaultTimeoutValues map[string]Upstream = map[string]Upstream{
 		HTTPClientTLSHandshakeTimeout:   time.Second * 5,
 		HTTPClientResponseHeaderTimeout: time.Second * 5,
 	},
+	ALERTMANAGER: {
+		HTTPClientTimeout:               time.Second * 10,
+		HTTPClientDialerTimeout:         time.Second * 3,
+		HTTPClientTLSHandshakeTimeout:   time.Second * 4,
+		HTTPClientResponseHeaderTimeout: time.Second * 3,
```

Comment:
```suggestion
		HTTPClientTimeout:               time.Second * 15,
		HTTPClientDialerTimeout:         time.Second * 5,
		HTTPClientTLSHandshakeTimeout:   time.Second * 5,
		HTTPClientResponseHeaderTimeout: time.Second * 5,
```
nit: It's behavior similar to the distributor